keyboards/coseyfannitutti/discipline/rules.mk

```diff
@@ -38,4 +38,4 @@ AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs
 
-LAYOUTS = 65_ansi
+LAYOUTS = 65_ansi 65_iso
```

Comment:
For Community Layout support to work, all the supporting keyboards' layout macros need to take their keycodes in the same order.

https://github.com/qmk/qmk_firmware/compare/85807b07569a67597f12977f630dce50db6a3dc5...486fb16ee2a4d004f0937f83411d4513fd1c31f7

Reply:
Okay, updated the macros, but haven't yet touched the via layout. I'm using 65_iso, essentially as a LAYOUT_all with ANSI layout as the default keymap for the VIA layout.

ANSI backslash is electrically tied to ISO enter, and ANSI enter to ISO hash.

Because this doesn't support extrakey, I wanted users to still have a clear layout on qmk config with an ISO enter in the .json so I left it as 65_iso instead of all. Is there a better way to go about that?

Reply:
Ah okay, go ahead and leave it as LAYOUT_65_iso.